<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creation Mode</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QLineEdit" name="dz_box"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ΔZ</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ΔY</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dx_box"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ΔX</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="dy_box"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="structure_combobox">
       <item>
        <property name="text">
         <string>Pipe + Bend</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pipe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Valve</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Expansion Joint</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reducer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circular Beam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangular Beam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I Beam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>T Beam</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C Beam</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connect_button">
       <property name="text">
        <string>Connect Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="section_button">
         <property name="text">
          <string>Default Section</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="material_button">
         <property name="text">
          <string>Default Material</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="apply_button">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dx_box</tabstop>
  <tabstop>dy_box</tabstop>
  <tabstop>dz_box</tabstop>
  <tabstop>section_button</tabstop>
  <tabstop>material_button</tabstop>
  <tabstop>apply_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
